internal/controllers/device_config_controller.go

```diff
@@ -7,79 +7,37 @@ import (
 	"fmt"
 	"time"
 
-	pbuser "github.com/DIMO-Network/shared/api/users"
 	"github.com/DIMO-Network/vehicle-signal-decoding/internal/config"
+	"github.com/DIMO-Network/vehicle-signal-decoding/internal/core/services"
 	"github.com/DIMO-Network/vehicle-signal-decoding/internal/infrastructure/db/models"
 	"github.com/DIMO-Network/vehicle-signal-decoding/pkg/grpc"
 	"github.com/gofiber/fiber/v2"
 	_ "github.com/lib/pq" //nolint
 	"github.com/pkg/errors"
 	"github.com/rs/zerolog"
-	"github.com/volatiletech/sqlboiler/v4/boil"
 	"github.com/volatiletech/sqlboiler/v4/queries/qm"
 	"google.golang.org/protobuf/encoding/protojson"
 	"google.golang.org/protobuf/proto"
 )
 
 type DeviceConfigController struct {
-	Settings    *config.Settings
-	log         *zerolog.Logger
-	db          *sql.DB
-	usersClient pbuser.UserServiceClient
+	Settings      *config.Settings
```

Comment:
```suggestion
	settings      *config.Settings
```
settings aren't being exported so no need to have it start with capital S